python/ray/experimental/state.py

```diff
@@ -321,7 +322,11 @@ def _task_table(self, task_id):
             "ActorCreationDummyObjectID": (
                 task.actor_creation_dummy_object_id().hex()),
             "ActorCounter": task.actor_counter(),
-            "Args": task.arguments(),
+            # For Java task, pickle will fail to load a data
+            # to Java object in `task.arguments()`.
+            "Args": (task.arguments()
+                     if task.language() == ray.gcs_utils.Language.PYTHON else
+                     ["<java-argument>"]),
```

Comment:
I think we should fix this issue inside the `arguments` method, otherwise if someone else calls `arguments`, it will also crash.